python/ray/train/tensorflow/tensorflow_trainer.py

```diff
@@ -45,6 +45,16 @@ def train_loop_per_worker(config: Dict):
     Inside the ``train_loop_per_worker`` function, you can use any of the
     :ref:`Ray AIR session methods <air-session-ref>`.
 
+    .. warning::
+        Ray will not automatically set any environment variables or configuration
+        related to local parallelism / threading
+        :ref:`aside from "OMP_NUM_THREADS" <omp-num-thread-note>`.
+        If you desire greater control over TensorFlow threading, use
+        the ``tf.config.threading`` module (eg.
+        ``tf.config.threading.set_inter_op_parallelism_threads(num_cpus)``)
+        at the beginning of your ``train_loop_per_worker`` function.
```

Comment:
Can we move this to a TensorFlow user guide instead?

Reply:
Will that have enough visibility?

Reply:
I've put it in both places